assets/js/alerts.js

```diff
@@ -184,8 +184,9 @@ function populateAlertTable(alerts) {
       const generatorLink = alert.generatorURL
         ? `<a href="${alert.generatorURL}" class="statusTileHealth" target="_blank">View</a>`
         : 'N/A'
+      const silencedAlert = alert.status.silencedBy.length > 0 ? 'silenced-alert' : 'active-alert'
       $('#statusRows')
-        .append(`<tr data-alert-name="${alert.labels.application}" data-environment="${alert.labels.application}" data-environment-type="${alert.labels.environment}" data-silenced="${alert.status.state}" id="tile-${alert.labels.application}-${alert.labels.environment}">
+        .append(`<tr class=${silencedAlert} data-alert-name="${alert.labels.application}" data-environment="${alert.labels.application}" data-environment-type="${alert.labels.environment}" data-silenced="${alert.status.state}" id="tile-${alert.labels.application}-${alert.labels.environment}">
```

Comment:
Still need double quotes around the class name in the tr `<tr class="${silencedAlert}"`